doc/developer-guide/api/functions/TSIpStringToAddr.en.rst

```diff
@@ -53,4 +49,3 @@ Notes
 =====
 
 This API may be changed in the future version since it is experimental.
-
```

Comment:
Removing this unnecessary blank line seems unrelated to this PR, should be in it's own PR.

Reply:
Brutal

Reply:
Welcome to my world.